Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -2066,3 +2076,22 @@ extension StatusTableViewController: ServicesViewModelDelegate {
     }
 
 }
+
+extension StatusTableViewController: PreferredGlucoseUnitPublisher {
+    func addPreferredGlucoseUnitObserver(_ observer: PreferredGlucoseUnitObserver, queue: DispatchQueue = .main) {
```

Comment:
Why are we maintaining a separate list of glucose observers here rather than just adding the observer to DeviceDataManager's queue? I think this should be cleaned up.

Reply:
The intent was that the device manager just handled devices and this VC was to handle UI needs. But I'm not against putting everything in the device manager.